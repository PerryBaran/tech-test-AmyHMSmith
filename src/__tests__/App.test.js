import React from "react";
import { render, screen, waitFor } from '@testing-library/react';
import App from "../../src/components/App";

describe("App", () => {
  test("renders App component correctly", async () => {
    // jest.spyOn(axios, "get").mockResolvedValue({ data: forecast });

    const { asFragment } = render(<App />);

    expect(asFragment()).toMatchSnapshot();
  });
});