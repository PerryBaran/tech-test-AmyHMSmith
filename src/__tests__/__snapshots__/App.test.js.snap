// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders App component correctly 1`] = `
<DocumentFragment>
  <div
    class="app"
  >
    <img
      alt="nasaLogo"
      class="nasa-logo"
      src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
    />
    <form
      class="search-form"
    >
      <input
        class="search-input"
        type="text"
      />
      <button
        class="search-button"
        type="submit"
      >
        Go!
      </button>
    </form>
    <div
      class="no-results"
    >
      <p>
        No results
      </p>
    </div>
  </div>
</DocumentFragment>
`;
